internal/dynamicanalysis/analysis.go

```diff
@@ -28,6 +32,24 @@ var resultError = &Result{
 	},
 }
 
+func ParseURLsFromSSlStripOutput(content []byte) []string {
```

Comment:
Do you think it is worth de-duping URLs?

Reply:
Maybe we could do this further downstream? (i.e. during reporting) - having all the original data could be useful for some cases

Reply:
it's very useful to have the raw data as it will allow us to analyze it and search for data such as exfiltrated secrets for example, nevertheless, we ought to have the URLS in the report as some addresses are proven IOC's.